src/powerquery-language-services/analysis/analysisBase.ts

```diff
@@ -163,6 +167,30 @@ export abstract class AnalysisBase implements Analysis {
         );
     }
 
+    private static isValidHoverIdentifier(activeNode: PQP.Inspection.ActiveNode): boolean {
+        const ancestry: ReadonlyArray<PQP.Parser.TXorNode> = activeNode.ancestry;
+        if (ancestry.length <= 1) {
+            return true;
+        }
+
+        const leaf: PQP.Parser.TXorNode = PQP.Assert.asDefined(ancestry[0]);
```

Comment:
Remind me what a `GeneralizedIdentifier` is again?

Reply:
Generalized identifiers are identifiers that can have spaces between them and don't restrict keyword usage. Most commonly they are field names when dealing with records/tables, eg.

`table[a multi word identifier]`
`table[date]`